packages/components/src/truncate/types.ts

```diff
@@ -17,19 +29,22 @@ export interface Props {
 	 *
 	 * @default 'auto'
 	 */
-	ellipsizeMode?: TruncateEllisizeMode;
+	ellipsizeMode?: TruncateEllipsizeMode;
 	/**
 	 * Determines the max characters when `truncate` is set.
```

Comment:
The description for `limit` is not very clear to me, and doesn't seem to represent the way the component currently works — maybe we can improve it? Something like

_Determines the max number of characters to be displayed before the rest of the text gets truncated. Requires `ellipsizeMode` to assume values different from `auto` and `none`._